oras/auth/base.py

```diff
@@ -120,37 +133,3 @@ def set_basic_auth(self, username: str, password: str):
         :type password: str
         """
         self._basic_auth = auth_utils.get_basic_auth(username, password)
-
-    def request_anonymous_token(self, h: auth_utils.authHeader, headers: dict) -> bool:
```

Comment:
To my understanding this is dead code and completely incompatible with the overriding method in token auth, which is actually being used.